os/kernel/debug/sysdbg.c

```diff
@@ -421,53 +428,69 @@ static ssize_t sysdbg_write(FAR struct file *filep, FAR const char *buffer, size
 
 	sscanf(buffer, "%s %s", arg1, arg2);
 
-	if (strncmp("enable_monitor", arg1, 14) == 0) {
+	if (strncmp("help", arg1, 5) == 0) {
+		show_usage();
+		return 0;
```

Comment:
It needs to return `-1` by line 416, `ret - 0 indicates success and -1 indicates failure`.

Reply:
Returning -1 is not desirable here as it is user intention to request for help.